main.go

```diff
@@ -102,16 +103,20 @@ func main() {
 
 	//
 	// Fastlane session
-	fmt.Println()
-	log.Infof("Ensure cookies for Apple Developer Portal")
-
-	fs, errors := devportalservice.SessionData()
-	if errors != nil {
-		log.Warnf("Failed to activate the Bitrise Apple Developer Portal connection: %s\nRead more: https://devcenter.bitrise.io/getting-started/connecting-apple-dev-account/ \nerrors:")
-		for _, err := range errors {
-			log.Errorf("%s\n", err)
+	fs, err := devportalservice.SessionData()
+	if err != nil {
+		if networkErr, ok := err.(devportalservice.NetworkError); ok && networkErr.Status == http.StatusNotFound {
+			log.Debugf("")
```

Comment:
Empty line could be printed once, before the `if err != nil`

Reply:
but in case of NotFound Network error, it is just a debug log, otherwise a normal log.